### ISSUE TITLE
[BUG] Onboarding broken at chat redirect

### ISSUE THREAD

**Issue by tfmertz** [anthropics/claude-code#25056]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

After trying to login to claude code using Google Login I get sent to the "Authorize" page, I see the Authorize button, but quickly get redirected back to /onboarding, which sits indefinitely at a loading with multiple browser errors.

I've tried logging out and back in and using multiple browsers, but always get stuck at the same place, it basically keeps restarting the onboarding process.

<img width="2299" height="1717" alt="Image" src="https://github.com/user-attachments/assets/c0f76148-7617-43c8-bc04-0253117cdb1e" />

### What Should Happen?

The app should redirect and let me authorize claude code.

### Error Messages/Logs

```shell
`DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.

If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.

For more information, see https://radix-ui.com/primitives/docs/components/dialog 92351-356b863509866654.js:13:207
[REACT_QUERY_CLIENT] QueryClient error: Error: organization_uuid: Extra inputs are not permitted
    NextJS 3
92351-356b863509866654.js:13:207
Error sending segment performance metrics TypeError: NetworkError when attempting to fetch resource.
    o analytics.min.js:1
    c remote-metrics.ts:138
    u tslib.es6.js:103
    u tslib.es6.js:108
    u tslib.es6.js:80
    u tslib.es6.js:72
    send analytics.min.js:1
    c remote-metrics.ts:125
    u tslib.es6.js:103
    u tslib.es6.js:108
    u tslib.es6.js:80
    u tslib.es6.js:72
    flush analytics.min.js:1
    l remote-metrics.ts:83
remote-metrics.ts:53:11
Content-Security-Policy: The page’s settings blocked the loading of a resource (connect-src) at https://a-api.anthropic.com/v1/m because it violates the following directive: “connect-src 'self' https://api.segment.io https://*.segment.io https://*.segment.com https://a-cdn.anthropic.com https://*.google.com https://*.facebook.com https://*.facebook.net https://*.doubleclick.net”
```

### Steps to Reproduce

1. Create a new account on claude code using a Google Account
2. Try to complete onboarding in a browser
3. Get's stuck after claude finishes asking questions and tries to redirect to the chat page.

### Claude Model

None

### Is this a regression?

I don't know

### Claude Code Version

2.1.39 (Claude Code)   

### Platform

Anthropic API

### Operating System

Windows

### Terminal/Shell

WSL (Windows Subsystem for Linux)

**Comment by rranzmaier** [comment]
Operating System
Linux (CachyOs)

I tried different browsers  Chome, Firefox, Chromium and Edge.

Same Issue:

`92351-356b863509866654.js:13   POST https://statsig.anthropic.com/v1/rgstr?k=client-U9UUxwLKFe7SJXyDnolyBR2xZtEP8m1pN2uptqkJ5qP&st=javascript-client-react&sv=3.12.1&t=1770838420338&sid=8d1f442b-ffb4-4412-bfb5-a15ccb242eb1&ec=40 net::ERR_CONNECTION_REFUSED
(anonymous) @ 92351-356b863509866654.js:13
(anonymous) @ 66320-81a87d78a2d04619.js:1
with @ 65279-4a359f73205c3ec9.js:1
t.with @ 57151-f2f3c85ef9bf2118.js:1
(anonymous) @ 66320-81a87d78a2d04619.js:1
(anonymous) @ 66320-81a87d78a2d04619.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendRequest @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
o @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendRequest @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
o @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendRequest @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
o @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendRequest @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
a @ 53396-f7a920260027d075.js:1
Promise.then
l @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
post @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendEventsViaPost @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_sendEvents @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_retryFailedLogs @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_capture @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
_safeRetryFailedLogs @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
(anonymous) @ 53396-f7a920260027d075.js:1
r @ 92351-356b863509866654.js:10
92351-356b863509866654.js:13   ERROR  [Statsig] A networking error occurred during POST request to https://statsig.anthropic.com/v1/rgstr?k=client-U9UUxwLKFe7SJXyDnolyBR2xZtEP8m1pN2uptqkJ5qP&st=javascript-client-react&sv=3.12.1&t=1770838420338&sid=8d1f442b-ffb4-4412-bfb5-a15ccb242eb1&ec=40. TypeError: Failed to fetch (statsig.anthropic.com) TypeError: Failed to fetch (statsig.anthropic.com)
    at 92351-356b863509866654.js:13:1635
    at 66320-81a87d78a2d04619.js:1:110222
    at G.with (65279-4a359f73205c3ec9.js:1:113353)
    at t.with (57151-f2f3c85ef9bf2118.js:1:4503)
    at 66320-81a87d78a2d04619.js:1:110127
    at new Promise (<anonymous>)
    at 66320-81a87d78a2d04619.js:1:110100
    at a.<anonymous> (53396-f7a920260027d075.js:1:38173)
    at Generator.next (<anonymous>)
    at a (53396-f7a920260027d075.js:1:35382)`

**Comment by peterBrxwn** [comment]
Mine has finally worked.
